styleguide/derek/global-components/filter-bar/filter-bar.js

```diff
@@ -17,35 +67,27 @@
 
     // Callback to check if inbounds of side menu
     function checkInBounds(wt) {
```

Comment:
Can you do me one more favor on this function? 

The only time window top is calculated is right before the function is called.  Meaning, there's zero difference in passing that value in, versus just calculating it inside the function.  
```
          const $rsWindowTop = parseInt($rsWindow.scrollTop(), 10);
          inBounds = checkInBounds($rsWindowTop);
```

Can we just get rid of the parameter, and then change the array item below to this?:

```
windowTop: $rsWindow.scrollTop(),
```